### PR TITLE
Feature/qap 1700 continue on error ee platform suites

### PR DESCRIPTION
**Pull request by diasdm**
Changes:
* Set continue of failure for all QA suites github action step
* Add step to fail pipeline if any QA suite failed which had not continue on failure parameter set to true
* Add parameters for flow tests

These changes improve QA experience by:
* running all suites before failing pipeline
* allow QA members to temporarily set stages to continue on failure without having to change the pipeline

These changes are not backwards compatible. Do we accept the compatibility break?

### CODE REVIEW

File: .github/workflows/integration-build-platform.yml [hunk 1]
```diff
@@ -428,11 +480,42 @@ jobs:
           pytest -s \
             --movai-user ${{ steps.install.outputs.movai_user }} \
             --movai-pw ${{ steps.install.outputs.movai_pwd }} \
-            -m 'not simulator and not exposed_ros' \
+            -m '${{ steps.flow_tests_setup.outputs.pytest_markexpr }}' \
             --tb=short
           deactivate
           rm -R venv
 
+      - name: Fail execution if tests failed
```
Comment (almeidamov) [inline .github/workflows/integration-build-platform.yml]: QA tests*

**Review by almeidamov — CHANGES_REQUESTED** [review]

**Review by duartecoelhomovai — CHANGES_REQUESTED** [review]
We can do parallel stages if they do not have dependencies between them.
I can help you architecture'ize it.
Request changes is for myself.

**Review by almeidamov — APPROVED** [review]

**Comment by AlexFernandes-MOVAI** [comment]
@diasdm and @duartecoelhomovai please review conflicts

**Comment by diasdm** [comment]
Due to the tests now running in parallel, there is less motivation to have this

**Pull request closed without merging**